notebooks/Forecast_skill.ipynb

```diff
@@ -0,0 +1,1107 @@
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>2023-01-07</th>\n",
+       "      <td>1</td>\n",
+       "      <td>13804.10</td>\n",
+       "      <td>12663.04</td>\n",
+       "      <td>14319.96</td>\n",
+       "      <td>15237.57</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2023-01-07</th>\n",
+       "      <td>2</td>\n",
+       "      <td>13302.96</td>\n",
+       "      <td>12898.16</td>\n",
+       "      <td>14319.96</td>\n",
+       "      <td>15237.57</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2023-01-07</th>\n",
+       "      <td>3</td>\n",
+       "      <td>13388.44</td>\n",
+       "      <td>13505.24</td>\n",
+       "      <td>14319.96</td>\n",
+       "      <td>15237.57</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2023-01-07</th>\n",
+       "      <td>4</td>\n",
+       "      <td>13014.56</td>\n",
+       "      <td>13444.72</td>\n",
+       "      <td>14319.96</td>\n",
+       "      <td>15237.57</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2023-01-07</th>\n",
+       "      <td>5</td>\n",
+       "      <td>12845.50</td>\n",
+       "      <td>13086.09</td>\n",
+       "      <td>14319.96</td>\n",
+       "      <td>15237.57</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2023-01-07</th>\n",
+       "      <td>6</td>\n",
+       "      <td>12801.62</td>\n",
+       "      <td>13413.75</td>\n",
+       "      <td>14319.96</td>\n",
+       "      <td>15237.57</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2023-01-07</th>\n",
+       "      <td>7</td>\n",
+       "      <td>13003.76</td>\n",
+       "      <td>13204.51</td>\n",
+       "      <td>14319.96</td>\n",
+       "      <td>15237.57</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "            lead_time   model_1   model_2  observation     dummy\n",
+       "date                                                            \n",
+       "2023-01-07          1  13804.10  12663.04     14319.96  15237.57\n",
+       "2023-01-07          2  13302.96  12898.16     14319.96  15237.57\n",
+       "2023-01-07          3  13388.44  13505.24     14319.96  15237.57\n",
+       "2023-01-07          4  13014.56  13444.72     14319.96  15237.57\n",
+       "2023-01-07          5  12845.50  13086.09     14319.96  15237.57\n",
+       "2023-01-07          6  12801.62  13413.75     14319.96  15237.57\n",
+       "2023-01-07          7  13003.76  13204.51     14319.96  15237.57"
+      ]
+     },
+     "execution_count": 102,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pmr.index.name = \"date\"\n",
+    "matched_model_results = matched_model_results.merge(pmr, on=\"date\", how=\"inner\")\n",
```

Comment:
Consider to use [`pandas.DataFrame.join`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.join.html) to join on index instead of `merge`.